docs/noise.ipynb

```diff
@@ -384,7 +383,7 @@
     "id": "67c8ddbc10f3"
    },
    "source": [
-    "For concrete examples, `cirq.X`, `cirq.BitFlipChannel`, and `cirq.AmplitudeDampingChannel` are all subclasses of `cirq.Gate`.\n",
+    "For concrete examples, `cirq.X`, consider `cirq.BitFlipChannel`, and `cirq.AmplitudeDampingChannel` which are all subclasses of `cirq.Gate`.\n",
```

Comment:
This wording is awkward.  Should consider be before cirq.X?

Reply:
Good catch, thanks!